lms/djangoapps/email_marketing/migrations/0007_auto_20170803_0903.py

```diff
@@ -0,0 +1,28 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('email_marketing', '0006_auto_20170711_0615'),
+    ]
+
+    operations = [
+        migrations.RemoveField(
+            model_name='emailmarketingconfiguration',
+            name='sailthru_activation_template',
+        ),
+        migrations.AddField(
```

Comment:
I don't see a migration to move the value we have in `sailthru_activation_template` to `sailthru_welcome_template`. Should we add that migration so we don't miss it during deployment? Otherwise, we would've stopped all welcome email sending after this change is deployed out, and only recover that after we manually give the value to the new field.